src/main/kotlin/no/nb/bikube/catalogue/collections/model/dto/CollectionsItemDto.kt

```diff
@@ -54,14 +54,17 @@ data class AlternativeNumberInput (
     val type: String
 )
 
-fun createNewspaperItemDto(item: ItemInputDto, manifestationCatalogueId: String): ItemDto {
+fun createNewspaperItemDto(
+    item: ItemInputDto,
+    manifestationCatalogueId: String
+): ItemDto {
     val useUrn = item.digital == true && !item.urn.isNullOrBlank()
 
     return ItemDto(
-        title = item.title,
+        title = item.name,
         format = if (item.digital == true) CollectionsFormat.DIGITAL.value else CollectionsFormat.PHYSICAL.value,
         recordType = CollectionsRecordType.ITEM.value,
-        inputName = "Bikube API", // TODO: Change when we have authentication in place
+        inputName = "${item.username} (Bikube)",
```

Comment:
Det ligger et eget notefelt på både input og edit-gruppene (input.notes (mi) og edit.notes(mm)). Kanskje vi heller skulle brukt det for å markere at det er redigert/sendt inn via Bikube?

Reply:
Lurt!

Reply:
Vet ikke om vi får bruk for å hente ut denne informasjonen i APIet (pr. nå), så legger ikke det inn i den tekst-Item eller Title. Men har lagt den inn så det kommer ut fra Collections APIet og inn i CollectionsObject i bikube, selv om det ikke blir brukt. @fredrikmonsen